src/Tizen.NUI.Components/Style/AlertDialogStyle.cs

```diff
@@ -50,27 +51,27 @@ public AlertDialogStyle(AlertDialogStyle style) : base(style)
         /// Gets or sets the AlertDialog Title TextLabel style.
         /// This style is applied if AlertDialog TitleContent is a TextLabel.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
         public TextLabelStyle TitleTextLabel { get; set; } = new TextLabelStyle();
 
         /// <summary>
         /// Gets or sets the AlertDialog Message TextLabel style.
         /// This style is applied if AlertDialog Content is a TextLabel.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
         public TextLabelStyle MessageTextLabel { get; set; } = new TextLabelStyle();
 
         /// <summary>
         /// Gets or sets the AlertDialog ActionContent style.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
         public ViewStyle ActionContent { get; set; } = new ViewStyle();
 
         /// <summary>
         /// Style's clone function.
         /// </summary>
         /// <param name="bindableObject">The style that need to copy.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
if this override doesn't do anything special, then it's probably better to use "\<inheritdoc/\>"

Reply:
Thank you for your comment.
It has been updated.